index.html

```diff
@@ -426,16 +434,19 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
                 <div class="bannerbox" id="story">
                     <h3 class="bannerbox__title">Our Story</h3>
                     <div class="bannerbox__content">
-                        <p>Carpool Vote connects volunteer drivers with anybody who needs a ride to claim their vote.</p>
-                        <p>Many of our riders need a driver after losing full protection of the Voting Rights Act in 2013, as state governments no longer need permission to change election rules. The 2016 US Presidential election will be the first since this change has happened.</p>
-                        <p>Under the new rules, voters in several states face new barriers to getting to the polls. Some of the obstacles that Carpool Vote aims to reduce include:</p>
-                        <ul>
-                            <li>Polling stations face closure in lower-income communities of color,</li>
-                            <li>Would-be voters have to travel out of town to get a photo ID to register,</li>
-                            <li>The periods available to register and vote become shorter, and</li>
-                            <li>Requesting an absentee ballot or voting by mail can be difficult in many states.</li>
-                        </ul>
-                        <p>These rules - along with others introduced since the change - affect everybody but especially people of color, people with disabilities, young people, elderly people, and women.</p>
+                        <p>In many states, new voting rules have made it more difficult for people to get to polling stations. These rules particularly affect People of Color, People with Disabilities, Young People, Elderly People, and Women.Carpool Vote is a tool to fight back against an unfair political system in the 2016 election.</p>
+
+                        <p>We connect volunteer drivers with anybody who needs a ride - to the polls or voter I.D. office.</p>
+
+                        <button class="button" id="story-button"> <a href="/story" class="story"> MORE </a> </button>
```

Comment:
Instead of a link inside of a button, can we just make this a link with the button class on it? e.g. 
```
<p><a href="/story" class="button">Read more</a></p>
```